npm/pkg/dataplane/policies/policy_windows.go

```diff
@@ -100,9 +105,11 @@ func (acl *ACLPolicy) convertToAclSettings() (*NPMACLPolSettings, error) {
 	policySettings.LocalAddresses = srcListStr
 	policySettings.RemoteAddresses = dstListStr
 	policySettings.RemotePorts = dstPortStr
+	policySettings.LocalPorts = ""
```

Comment:
It looks unnecessary. policySettings has default value (empty string) for string.

Reply:
Just a preventive measure, can remove it.